javascript/javascript1/week3/travelInfo.js

```diff
@@ -0,0 +1,10 @@
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+  function findTravelTime(travelInformarion){
+      let time = travelInformarion.destinationDistance/travelInformation.speed;
+       return `${Math.floor(time)} hours and ${Math.floor((time-Math.floor(time)) * 60)} minutes`;
```

Comment:
Nice!

I'd assign these to their own variables to make it easier to read:
```javascript
const hours = Math.floor(time);
const minutes = Math.floor((time - hours) * 60);

return `${hours} hours and ${minutes} minutes`;
```

Reply:
yes, thats awesome ! i will rewrite it.